packages/theme-docs/src/i18n/es-ES.js

```diff
@@ -1,6 +1,7 @@
 export default {
   search: {
-    placeholder: 'Buscar en documentos (Pulsa "/" para afinar)'
+    placeholder: 'Buscar en documentos (Pulsa "/" para enfocar)',
+    Searching: 'Buscando'
```

Comment:
I would change it to lowercase `searching` as it's more consistent with other properties.

Reply:
Then apply _toUpperCase_ to the first letter when used on components? Good observation, I'll commit that change. 😉